CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.15)
+cmake_policy(SET CMP0091 NEW) # Use MSVC_RUNTIME_LIBRARY to select the runtime
+
+project(Verilator
+    VERSION 4.025.99
+    HOMEPAGE_URL https://verilator.org
+    LANGUAGES CXX
+)
+
+include(GNUInstallDirs)
```

Comment:
It would be nice if the cmake toolchain would also support install similar to autotools.
I don't think `GNUInstallDirs` is currently used.

Reply:
This new-ly cmake support can also generate a `verilator-config.cmake`. This would also mean that the current `verilator-config.cmake.in` should be modified (or split).

Reply:
I just read your first comment about this pr having no installation. So ignore.